src/manifold/src/edge_op.cpp

```diff
@@ -159,15 +176,18 @@ void Manifold::Impl::SimplifyTopology() {
   }
 #endif
 
-  for (const int edge : flaggedEdges) DedupeEdge(edge);
-
-  flaggedEdges.resize(halfedge_.size());
-  numFlagged =
-      copy_if<decltype(flaggedEdges.begin())>(
-          policy, countAt(0), countAt(halfedge_.size()), flaggedEdges.begin(),
-          ShortEdge({halfedge_.cptrD(), vertPos_.cptrD(), precision_})) -
-      flaggedEdges.begin();
-  flaggedEdges.resize(numFlagged);
+  // --------------------------------------------
+  for (int i = 0; i < numFlagged; ++i) DedupeEdge(flaggedEdges[i]);
```

Comment:
Would it be simpler to skip the `flaggedEdges` entirely and combine the loops? We can just build in a counter and put the debug output after instead of before.

Reply:
Sounds possible.